role-management-commands/pause-role.md

```diff
@@ -22,14 +25,20 @@ For more information on editing permission requirements for specific users/roles
 
 ### Options
 
-* `user`: <TODO: describe>
-* `role`: <TODO: describe>
-* `duration`: <TODO: describe>
-* `reason`: <TODO: describe>
+* `user`: The user to pause the role for. You can mention the user or use their ID to specify the user.
+* `role`: The role to pause for the user. You can mention the role or use its ID to specify the role.
+* `duration`: The duration for which the role should be paused for the user. You can specify the duration in the
```

Comment:
Explanation on duration formats is already in another page (see https://moonlightbot.gitbook.io/docs/start-up/arguments), putting it here us too verbose